### PR TITLE
in-app messaging updates

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

File: src/pages/documentation/iam/tabs/setup.md [hunk 1]
```diff
@@ -14,11 +14,8 @@ allprojects {
 } 
 ```
 
-The Messaging extension has a dependency on the Optimize extension. The Optimize extension must be added as a dependency in the app level Gradle file.
-
 ```groovy
-implementation('com.adobe.marketing.mobile:messaging:1.2.0-beta-1-SNAPSHOT')
-implementation('com.adobe.marketing.mobile:optimize:1.0.0-SNAPSHOT')
+implementation('com.adobe.marketing.mobile:messaging:1.3.0-beta-1-SNAPSHOT')
```
Comment (rymorale) [inline src/pages/documentation/iam/tabs/setup.md:18]: this should be `1.3.0-beta-3-SNAPSHOT`

**Review by rymorale — APPROVED** [review]
looks good, just need to update the android messaging staging build version

File: src/pages/documentation/iam/public-classes.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ iOS
 
 Signals to the UIService that the message should be removed from the UI.
 
-If `autoTrack` is true, calling this method will result in an "inapp.dismiss" Edge Event being dispatched.
+If `autoTrack` is true, calling this method will result in an "decisioning.propositionDismiss" Edge Event being dispatched.
```
Comment (calebk1) [inline src/pages/documentation/iam/public-classes.md:75]: Same comment here - might be better to wrap this in a `` block as well.

File: src/pages/documentation/iam/tabs/public-classes.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ public void dismiss(final boolean suppressAutoTrack)
 
 ##### Parameters
 
-* *suppressAutoTrack* - if set to `true`, the "inapp.dismiss" Edge Event will not be sent regardless of the `autoTrack` setting.
+* *suppressAutoTrack* - if set to `true`, the "decisioning.propositionDismiss" Edge Event will not be sent regardless of the `autoTrack` setting.
```
Comment (calebk1) [inline src/pages/documentation/iam/tabs/public-classes.md:77]: Same here. Actually yeah, let's wrap the things in "" here with `` instead.

File: src/pages/documentation/iam/tabs/public-classes.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ public func dismiss(suppressAutoTrack: Bool? = false)
 
 ###### Parameters
 
-* *suppressAutoTrack* - if set to `true`, the "inapp.dismiss" Edge Event will not be sent regardless of the `autoTrack` setting.
+* *suppressAutoTrack* - if set to `true`, the "decisioning.propositionDismiss" Edge Event will not be sent regardless of the `autoTrack` setting.
```
Comment (calebk1) [inline src/pages/documentation/iam/tabs/public-classes.md:89]: Same here!

File: src/pages/documentation/iam/tutorials/javascript-from-native.md [hunk 5]
```diff
@@ -38,4 +38,11 @@ Android
...
-<Tabs query="platform=ios&task=call"/>
\ No newline at end of file
+<Tabs query="platform=ios&task=call"/>
+
+## Examples
+
+The test apps in this repository demonstrate executing JavaScript code from an in-app message's webview:
+
+- [Swift](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoApp)
+- [Objective-c](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoAppObjC)
```
Comment (calebk1) [inline src/pages/documentation/iam/tutorials/javascript-from-native.md:48]: This should be Objective-C, not Objective-c.

File: src/pages/documentation/iam/tutorials/messaging-delegate.md [hunk 6]
```diff
@@ -64,6 +64,13 @@ iOS
 
 <Tabs query="platform=ios&function=integrate-message"/>
 
+## Examples
+
+The test apps in this repository demonstrate using a `MessagingDelegate`:
+
+- [Swift](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoApp)
+- [Objective-c](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoAppObjC)
```
Comment (calebk1) [inline src/pages/documentation/iam/tutorials/messaging-delegate.md:72]: Same comment here! Should be Objective-C, not Objective-c.

File: src/pages/documentation/iam/tutorials/native-from-javascript.md [hunk 7]
```diff
@@ -61,3 +61,10 @@ When the user clicks the button inside of this in-app message, the handler confi
 ```text
 JavaScript body passed to native callback: native callbacks are cool!
 ```
+
+## Examples
+
+The test apps in this repository demonstrate calling native code from JavaScript:
+
+- [Swift](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoApp)
+- [Objective-c](https://github.com/adobe/aepsdk-messaging-ios/tree/staging/TestApps/MessagingDemoAppObjC)
```
Comment (calebk1) [inline src/pages/documentation/iam/tutorials/native-from-javascript.md:70]: Same comment wrt Objective-C.

**Review by calebk1 — COMMENTED** [review]
Looks good! Just a few minor comments.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**